package.json

```diff
@@ -77,6 +77,13 @@
     "get-next-version": "./tooling/index.js version next --always",
     "deps:check": "./tooling/index.js exec -- dependency-check ./package.json",
     "deps:generate": "./deps.js",
+    "build:samples": "echo 'Deprecated. Please use npm run samples:build'; exit 1",
+    "serve:samples": "echo 'Deprecated. Please use npm run samples:serve'; exit 1",
+    "test:samples": "echo 'Deprecated. Please use npm run samples:test'; exit 1",
```

Comment:
Can these three deprecated commands be removed?

Reply:
I may just make them interchangeable